build-utils.js

```diff
@@ -3,22 +3,16 @@ var RE = {
   version: /^(\d{4})(\d{2})(\d{2})$|^(\d+)(?:_(\d+))?(?:_(\d+))?$/
 };
 
-module.exports = buildEnvsTree;
+exports.parseEnvsVersions = parseEnvsVersions;
```

Comment:
tbh i'd much rather move these functions to separate files than to pretend that CJS has more than a default export.

Reply:
Back to the `scripts` folder then?

Reply:
i'm ok with that